### ISSUE TITLE
Password Management Support

### ISSUE THREAD

**Issue by ajmarroquin** [joelekstrom/fastmate#50]
Hi,

I've started using Fastmate recently as I enjoy having it be its own window, and native OS notifications but was wondering if there would be any plan for support with password managers. My email password is complex, and having to type it in regularly once I've been logged out is tedious. I'm wondering if there is a way to tie extensions into the app and use a password manager extension. I use 1password for reference.

**Comment by joelekstrom** [comment]
Hi! Thanks for using Fastmate 😃

For starters, you should be able to copy and paste the password from 1Password. Does this not work?

But yeah, 1Password integration should be possible. I'll add a todo to look into that, but I can't promise anything unfortunately!

**Comment by ajmarroquin** [comment]
@joelekstrom copy/pasting DOES work, my password is complex but memorable so I've just been doing it the old fashioned way. Just wanted to open the dialogue of potential improvement.

**Comment by pro-sumer** [comment]
You could use [Keyboard Maestro](http://www.keyboardmaestro.com/main/) and the [1Password CLI](https://support.1password.com/command-line/) to automate this for you.

(I do)

**Comment by mdbraber** [comment]
Wondering if implementing password support would be really worth it, considering e.g. 1Password 8 now offers a system-wide UI for very quickly copying passwords into apps. The tip from @pro-sumer seems like a good workaround if you'd really want to automate it.

**Comment by joelekstrom** [comment]
I agree it might be a bit unnecessary. I imagine most people using Fastmate stay logged in instead of logging out often (based on my own usage at least!)

**Comment by pro-sumer** [comment]
1Password 8's Universal Autofill did initially work for a while with Fastmate, but the last couple of weeks it seems to be broken?
